src/Symfony/Component/Translation/Tests/DependencyInjection/TranslatorPassTest.php

```diff
@@ -9,10 +9,10 @@
  * file that was distributed with this source code.
  */
 
-namespace Symfony\Bundle\FrameworkBundle\Tests\DependencyInjection\Compiler;
+namespace Symfony\Component\Translation\Tests\DependencyInjection;
```

Comment:
The original test case should be kept in `FrameworkBundle` with `@group legacy` and copied to the component, not moved.